netdata-installer.sh

```diff
@@ -208,6 +208,8 @@ while [ ! -z "${1}" ]; do
 		DONOTWAIT=1
 		shift 1
 	elif [ "$1" = "--auto-update" -o "$1" = "-u" ]; then
+		REINSTALL_COMMAND="${REINSTALL_COMMAND// -u/}"
```

Comment:
If we add an option in the future that starts with `-u`, this will not work. 
I suggest replacing it with the following:
```
REINSTALL_COMMAND="${REINSTALL_COMMAND/% -u/}";
REINSTALL_COMMAND="${REINSTALL_COMMAND/ -u[[:space:]]/ }"
```

This way, even the following will work:
`REINSTALL_COMMAND="test -u -unknown -u"; REINSTALL_COMMAND="${REINSTALL_COMMAND/% -u/}"; REINSTALL_COMMAND="${REINSTALL_COMMAND/ -u[[:space:]]/ }"; echo $REINSTALL_COMMAND`

Reply:
It is not me who is going to maintain that code so I went for the cleanest and most understandable code that works now. If you think that this needs to be written in a more defensive way then there is nothing stopping you from creating a competing PR.